lib/valkyrie/specs/shared_specs/persister.rb

```diff
@@ -31,7 +31,7 @@ class CustomResource < Valkyrie::Resource
     expect(saved.id).not_to be_blank
   end
 
-  it "does not save non-array properties" do
+  xit "does not save non-array properties" do
```

Comment:
Can this be `pending` with an explanation, or is this to be resolved in a forthcoming PR?

Reply:
I think I should probably just delete this spec.